common/script_values/babylon_infamy_multiplier.txt

```diff
@@ -0,0 +1,12 @@
+﻿babylon_infamy_multiplier = {
+	if = {
+		limit = {
+			infamy > 0
+		}
+		add = root.infamy
+		divide = 100
+	}
+	else = {
+		value = 0
+	}
```

Comment:
I would like to see some wiggle room at the lower infamy levels, maybe it starts from 15 or 20 instead of 0.